install_config/upgrading/upgrading_known_issues.adoc

```diff
@@ -0,0 +1,227 @@
+[[install-config-upgrading-known-issues]]
+= Known Issues
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+When upgrading your {product-title} cluster, there are two primary data
+migrations that take place. The first occurs while still running the current
+version of {product-title}, and the second happens after the upgrade completes.
+
+If the pre-upgrade migration fails the upgrade will be halted and the admin will
```

Comment:
fails, the upgrade   (both instances)
s/admin/administrator
s/will catalog/catalogs